tests/helpers.test.js

```diff
@@ -484,8 +521,27 @@ describe('Helper Tests', () => {
         expect(helpers.installDevDependencies.mock.calls[0][0]).not.toEqual('')
       })
     })
-    describe.skip('installAllPackagesToExistingProject', () =>  {
-
+    describe('installAllPackagesToExistingProject', () =>  {
+      beforeEach(() => {
+        store.dependencies = ''
+        store.devDependencies = ''
+      })
+      it("Calls installDependenciesToExistingProject if store contains dependencies", () => {
+        const mockFn = helpers.installDependenciesToExistingProject = jest.fn()
+        store.dependencies = 'react'
+        installAllPackagesToExistingProject()
+        expect(mockFn).toBeCalled()
+        expect(mockFn).toHaveBeenCalledWith('react')
+        expect(mockFn.mock.calls[0][0]).not.toEqual('')
```

Comment:
One of our engineers at BNR pointed out that we don't need to check the .not.toEqual as it's redundant